examples/gnrc_border_router/Makefile

```diff
@@ -7,6 +7,18 @@ BOARD ?= samr21-xpro
 # This has to be the absolute path to the RIOT base directory:
 RIOTBASE ?= $(CURDIR)/../..
 
+# Default to using ethos for providing the uplink when not on native
+UPLINK ?= ethos
```

Comment:
The comments reveals where this `UPLINK` variable becomes inconsistent :-/.

Reply:
True, I mean we have `spidev` support on `native`, together with native GPIOs (#12451) one could wire up an ethernet interface and a 802.15.4 radio to `native` on a raspberry pi.

But IMHO that's beyond the scope of this PR :wink: